plugins/modules/hana_backup.py

```diff
@@ -0,0 +1,253 @@
+#!/usr/bin/python
```

Comment:
does it really make sense to write a complete module for a few sql statements ? maybe it's less maintaining effort use the hdbsql module from sap_libs and write a role